app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -188,7 +190,11 @@ fun CreateActivityScreen(
           Spacer(modifier = Modifier.height(8.dp))
 
           ExposedDropdownMenuBox(
-              modifier = Modifier.testTag("chooseTypeMenu"),
+              modifier =
```

Comment:
Good job on calling these methods with the modifier to have aligned elements, and that fill the whole screen